src/Database/LSMTree/Normal.hs

```diff
@@ -294,22 +316,43 @@ open ::
        )
 open = undefined
 
+--TODO: we're missing an operation to delete a snapshot that is no longer
+-- required. This is necessary for management of disk space.
```

Comment:
Also listing snapshots?

Reply:
It's _probably_ not essential for most applications, but it's also a perfectly reasonable operation to provide, and plausibly useful.

Reply:
From operational perspective I think it's quite important. To be able to check what you think is there is actually there (and no more!).

Reply:
Agreed.